src/test/java/nl/hsac/fitnesse/util/iban/ATIbanTest.java

```diff
@@ -0,0 +1,65 @@
+package nl.hsac.fitnesse.util.iban;
+
+import nl.hsac.fitnesse.util.RandomUtil;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests AT Iban generator.
+ */
+public class ATIbanTest {
```

Comment:
The tests for each country specific generator seem to share a lot. Could you maybe extract a superclass for them?